sys/net/gnrc/network_layer/sixlowpan/frag/vrb/gnrc_sixlowpan_frag_vrb.c

```diff
@@ -80,6 +80,34 @@ gnrc_sixlowpan_frag_vrb_t *gnrc_sixlowpan_frag_vrb_add(
                                              vrbe->super.dst_len,
                                              addr_str), vrbe->out_tag);
             }
+            /* _equal_index() => append intervals of `base`, so they don't get
+             * lost. We use append, so we don't need to change base! */
+            else if (base->ints != NULL) {
+                gnrc_sixlowpan_frag_rb_int_t *tmp = vrbe->super.ints;
+
+                if (tmp != base->ints) {
+                    /* base->ints is not already vrbe->super.ints */
+                    if (tmp != NULL) {
+                        /* iterate before appending and check if `base->ints` is
+                         * not already part of list */
+                        while (tmp->next != NULL) {
+                            if (tmp == base->ints) {
+                                tmp = NULL;
+                            }
+                            /* cppcheck-suppress nullPointer
```

Comment:
@miri64 Can you confirm, just for my sake, why this comment is correct?

> tmp can't clearly be a NULL pointer here

If that statement is true then the `if` right above should never be taken.

Or did you perhaps forgot to add a `break`?

Reply:
Mhh... now that you say that, it seems as clear as day ^^". Yes, there is a break missing.

Reply:
See https://github.com/RIOT-OS/RIOT/pull/14683

Reply:
BTW Thanks to Coverity Scan

Reply:
Well, `cppcheck` also picked it up, I was just stupid ;-P